nservicebus/vscode.md

```diff
@@ -0,0 +1,282 @@
+
+## Prerequisities
+
+* This article assumes knowledge of NServiceBus solutions.
+* VS Code must have the [C# for Visual Studio Code (powered by OmniSharp)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) extension installed.
+
+## Configuration files
+
+The build and debug system in VS Code is controlled by two files located in a `.vscode` directory at the project root:
+
+* _{Project Root}_
+    * `.vscode/`
+        * `launch.json`
+        * `tasks.json`
+
+The **launch.json** file describes different projects within the solution and describes how to launch each one.
+
+The **tasks.json** file describes actions to take, the most common of which is to build the code. These tasks can be executed as a prerequisite of executing a configuration in **launch.json**.
+
+Both of these files can be generated from within VS Code using templates, and these are good starting points, but the templates will need tweaking for NServiceBus solutions.
+
+## tasks.json
+
+To create a starter template for this file:
+
+ 1. Attempt to start debugging.
+ 1. Click **Configure Task** in the error dialog that appears.
+ 1. Select **Create tasks.json from template**.
+ 1. Select **.NET Core**.
+ 
+ The template created by VS Code may work as-is if the project contains only one solution file in the project's root directory.
+
+The simplest **tasks.json** describes how to build the solution using `dotnet build`, only revealing the terminal if a problem is detected:
+
+```json
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "command": "dotnet build ${workspaceFolder}/path/to/SolutionName.sln",
+            "type": "shell",
+            "group": "build",
+            "presentation": {
+                "reveal": "silent"
+            }
+            "problemMatcher": "$msCompile"
+        }
+    ]
+}
+```
+
+For an NServiceBus solution, use the content above, and adjust the path of the `sln` file.
+
+For more information on **tasks.json**, see [Visual Studio Code: Integrate with External Tools via Tasks](https://code.visualstudio.com/docs/editor/tasks).
+
+## launch.json
+
+To create a starter template for this file:
+
+ 1. In the Debug toolbar, click the **No Configurations** dropdown.
+ 1. Select **Add Configuration**.
+ 1. Select **.NET Core**.
+
+The high-level structure of the launch.json file contains a collection of individual project objects in `configurations` and an optional [`compounds` collection](#launch-json-compounds) (not included in the template) that lists multiple configurations that should be launched at the same time.
```

Comment:
not included in the default file?

Or, perhaps since we've already stated that it's optional, we don't need the parenthetical phrase?